dask/dataframe/core.py

```diff
@@ -1029,23 +1040,21 @@ def elemwise(op, *args, **kwargs):
     else:
         op2 = op
 
-    if not all(df.divisions == dfs[0].divisions for df in dfs):
-        msg = 'All dask.Dataframe and dask.Series must have same divisions'
-        raise ValueError(msg)
-    if not all(df.npartitions == dfs[0].npartitions for df in dfs):
-        msg = 'All dask.Dataframe and dask.Series must have same npartitions'
-        raise ValueError(msg)
```

Comment:
I suggest that we add a `force=False` keyword argument.  This keeps people safe by default but also allows expert control.